code/modules/surgery/organs/filling_organ.dm

```diff
@@ -92,7 +92,7 @@
 				owner.adjust_nutrition(remove_amount) //since hunger factor is so tiny compared to the nutrition levels it has to fill
 			reagents.remove_reagent(reagent_to_make, (remove_amount*4)) //we consume our own reagents for food less efficently, allowing running out (may undo this multiplier later.)
 		else
-			if((reagents.total_volume < reagents.maximum_volume) && refilling) //if organ is not full.
+			if((reagents.total_volume < reagents.maximum_volume) && refilling && owner.nutrition > (NUTRITION_LEVEL_FED + 25)) //if organ is not full.
```

Comment:
Weird i thought the code already was like this.

Reply:
I don't know what happened with that.